content/en/examples/service/networking/default-ingressclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+  name: nginx
```

Comment:
We discussed possibly giving it a different name, but why shouldn't we provide the standard nginx example?

Reply:
Ah right. If I use the name `nginx`, people reading just the manifests can't easily spot which two names have to match. If I use another name, that only gets mentioned in two places, a lot more folks can infer that those two names must be the same.

Just `nginx` gets used all over the place&hellip;

Reply:
Okay, let's use `nginx-example`.